src/PhpSpreadsheet/Writer/Xls/Xf.php

```diff
@@ -514,15 +499,11 @@ private static function mapTextRotation($textRotation)
     private static function mapLocked($locked)
     {
         switch ($locked) {
-            case Protection::PROTECTION_INHERIT:
-                return 1;
-            case Protection::PROTECTION_PROTECTED:
-                return 1;
             case Protection::PROTECTION_UNPROTECTED:
                 return 0;
-            default:
-                return 1;
         }
+
+        return 1;
```

Comment:
It may be slightly less performant checking all cases; but I prefer their explicit use for readability.

Reply:
I had coded it the way I did not particularly for performance reasons, but rather for coverage reasons - there can sometimes be (and would in this case) an uncovered return statement (or even an uncovered fall out from the switch) if all possibilities are covered in the switch. I believe the way I have recoded it takes care of your readability requirement without leaving anything uncovered.